src/index.js

```diff
@@ -4,6 +4,19 @@ import createModal from './scripts/popup.js';
 const tvContainer = document.getElementById('tvContainer');
 const tvArray = [5, 169, 1871, 73, 51, 156];
 
+let responseArray = [];
+
+async function readLikes() {
+  const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Ql2WzJr90DiP5KlSpxzA/likes/', {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  const likes = await response.json();
+  responseArray = likes;
+}
+
 async function logTV(item, index) {
```

Comment:
To keep in mind the best practices with JavaScript I would suggest you make the functions as arrow functions 🏹